charts/neo4jv5-internal/values.preprod.yaml

```diff
@@ -0,0 +1,82 @@
+neo4j:
+  neo4j:
+    name: neo4jv5-internal
+    resources:
+      requests:
+        cpu: 1000m
+        memory: "2Gi"
+      limits:
+        cpu: 2000m
+        memory: "2Gi"
+
+    # Uncomment to set the initial password
+    # password: "my-initial-password"
+    passwordFromSecret: ""
+
+    # Uncomment to use enterprise edition
+    edition: "enterprise"
+    acceptLicenseAgreement: "yes"
+  config:
+    #dbms.connector.bolt.advertised_address: neo4j-pr-23.neo4j-preprod.k8s.dev.linkurious.net:7687
+    #dbms.connector.https.advertised_address: neo4j-pr-23.neo4j-preprod.k8s.dev.linkurious.net:443
+    # DATASETS: aml-2.0.0,crunchbase-2.0.0,fincrime-2.0.0
+    DATASETS: crunchbase-2.0.0
+    DATASET_NEO4J_VERSION: 5.3.0
+
+  podSpec:
+    initContainers:
+      - name: git-sync
+        image: k8s.gcr.io/git-sync/git-sync:v3.6.2
+        env:
+        - name: GIT_SYNC_REPO
+          value: https://github.com/Linkurious/docker-neo4j
+        - name: GIT_SYNC_DEST
+          value: docker-neo4j
+        - name: GIT_SYNC_BRANCH
+          value: feature/DN4J-11
+        - name: GIT_SYNC_ONE_TIME
+          value: 'true'
+        volumeMounts:
+          - name: data
+            mountPath: /tmp/git/
+      - name: neo4j-restore
+          # command: ['ls', '-lah', '/data/git-sync/', '&&', '/data/git-sync/docker-neo4j/scripts/dataset-download.sh']
+        command: ['/data/docker-neo4j/scripts/dataset-download.sh', '-p', '/data']
+        envFrom:
+          - secretRef:
+              name: neo4j-dev-nexus-dataset # imports NEXUS_TOKEN
```

Comment:
revert back to preprod

Reply:
just seeing this, what do you mean ?

Reply:
it's a note for myself, in neo4j-preprod namespace env the secret is neoj4-preprod ...
and we can't template this.